apps/meteor/app/api/server/lib/messages.ts

```diff
@@ -12,7 +27,7 @@ export async function findMentionedMessages({ uid, roomId, pagination: { offset,
 		throw new Error('invalid-user');
 	}
 
-	const cursor = await Messages.findVisibleByMentionAndRoomId(user.username, roomId, {
+	const cursor = await Messages.findVisibleByMentionAndRoomId((user as IUser).username, roomId, {
```

Comment:
What if you put this (as IUser) in line 25 instead? The type of `user` there should be IUser | null